tests/jobs/localisation/test_config1.py

```diff
@@ -0,0 +1,831 @@
+        ),
+    ],
+)
+def test_add_remove_obs_with_ref_point_and_field_scale_config(
+    obs_add, param_add, ref_point, method, range1, range2, angle, expected
+):
+    data = {
+        "log_level": 2,
+        "correlations": [
+            {
+                "name": "some_name",
+                "obs_group": {
+                    "add": [obs_add],
+                },
+                "param_group": {
+                    "add": [param_add],
+                },
+                "ref_point": ref_point,
+                "field_scale": {
+                    "method": method,
+                    "main_range": range1,
+                    "perp_range": range2,
+                    "angle": angle,
+                },
+            }
+        ],
+    }
+    conf = LocalisationConfig(observations=ERT_OBS, parameters=ERT_PARAM, **data)
+    assert len(conf.correlations) == 1
+    assert conf.correlations[0].obs_group.result_items == expected
+    assert conf.correlations[0].param_group.result_items == ["PARAM_FIELD1"]
+
+
+@pytest.mark.parametrize(
+    "obs_add, param_add, ref_point,  method, range1, " "range2, angle",
+    [
+        (
+            "OBS1*",
+            "PARAM_SURFACE1",
+            [550, 1050],
+            "gaussian_decay",
+            1700,
+            850,
+            310,
+        ),
+        (
+            "OBS1*",
+            "PARAM_SURFACE*",
+            [100, 150],
+            "exponential_decay",
+            0.1,
+            850,
+            0,
+        ),
+        (
+            "OBS1*",
+            "PARAM_SURFACE1",
+            [0, 750],
+            "exponential_decay",
+            700,
+            850,
+            100,
+        ),
+        (
+            "OBS1*",
+            "PARAM_SURFACE*",
+            [500, 750],
+            "exponential_decay",
+            1700,
+            500,
+            360,
+        ),
+        (
+            "OBS1*",
+            "PARAM_SURFACE*",
+            [500, 750],
+            "gaussian_decay",
+            0.1,
+            0.1,
+            0,
+        ),
+    ],
+)
+def test_add_remove_obs_with_ref_point_and_surface_scale_config(
+    obs_add, param_add, ref_point, method, range1, range2, angle
+):
+    data = {
+        "log_level": 2,
+        "correlations": [
+            {
+                "name": "some_name",
+                "obs_group": {
+                    "add": [obs_add],
+                },
+                "param_group": {
+                    "add": [param_add],
+                },
+                "ref_point": ref_point,
+                "surface_scale": {
+                    "method": method,
+                    "main_range": range1,
+                    "perp_range": range2,
+                    "angle": angle,
+                    "filename": "surface.txt",
+                },
+            }
+        ],
+    }
+    conf = LocalisationConfig(observations=ERT_OBS, parameters=ERT_PARAM, **data)
```

Comment:
This is not really asserting anything, what is the main purpose of the test?

Reply:
Most of the integration tests does not assert anything or just a few things. I have used the tests to verify that the loclisation config is created without errors and as a fast way to check that and find bugs when developing the code since it is faster than opening the ert gui and run the reek or drogon cases. The tests create the input data they need also.